lib/Sema/DeclSpec.cpp

```diff
@@ -1019,7 +1019,7 @@ bool DeclSpec::setFunctionSpecChecked(SourceLocation Loc,
   return false;
 }
 
-bool DeclSpec::setFunctionSpecForany(SourceLocation Loc,
+bool DeclSpec::setSpecForany(SourceLocation Loc,
```

Comment:
While you are at it, please fix the indentation for the arguments of this function.

Reply:
Done